links/plan.sh

```diff
@@ -0,0 +1,264 @@
+# This file is the heart of your application's habitat.
+# See full docs at https://www.habitat.sh/docs/reference/plan-syntax/
+
+# Required.
+# Sets the name of the package. This will be used in along with `pkg_origin`,
+# and `pkg_version` to define the fully-qualified package name, which determines
+# where the package is installed to on disk, how it is referred to in package
+# metadata, and so on.
+pkg_name=links
+
+# Required unless overridden by the `HAB_ORIGIN` environment variable.
+# The origin is used to denote a particular upstream of a package.
+pkg_origin=smacfarlane
```

Comment:
Yeah, I'm trying to figure out what I did there because that wasn't staged when I added these changes.  I think I'm going to abort this until I can undo whatever pre-coffee Scott did.